build.gradle

```diff
@@ -100,3 +100,33 @@ tasks.withType(Test) {
 
 applicationDefaultJvmArgs = ["-Xdebug", "-Xrunjdwp:transport=dt_socket,server=y,address=5005,suspend=n"]
 mainClassName = "io.ktor.server.netty.DevelopmentEngine"
+
+// Artifact publication configuration
+publishing {
+    publications {
+        ArtifactPublication(MavenPublication) {
+            from components.java
+            groupId project.group
+            artifactId "ktor-template-core"
+            version project.version
+        }
+    }
+}
+
+bintray {
+    user = project.properties["bintrayUser"]?.toString() ?: System.getenv("BINTRAY_USER")
+    key = project.properties["bintrayApiKey"]?.toString() ?: System.getenv("BINTRAY_API_KEY")
+    publications = ["ArtifactPublication"]
+    publish = true
+    pkg {
+        repo = "ktor-template"
+        name = "ktor-template-core"
```

Comment:
Package name is defined here along with its version. We also set the organization that holds all waterdog packages.